vocab/security/template.html

```diff
@@ -108,16 +131,40 @@ <h2>Namespaces</h2>
       </dl>
     </section>
 
-    <section id="class_definitions">
-      <h1>Class definitions</h1>
-    </section>
+    <section id="term_definitions">
+      <h1>Term definitions</h1>
 
-    <section id="property_definitions">
-      <h1>Property definitions</h1>
-    </section>
+      <section id="class_definitions" class="term_definitions">
+        <h2>Class definitions</h2>
+      </section>
+  
+      <section id="property_definitions" class="term_definitions">
+        <h2>Property definitions</h2>
+      </section>
+  
+      <section id="individual_definitions" class="term_definitions">
+        <h2>Definitions for individuals</h2>
+      </section>
+    </section>   
 
-    <section id="individual_definitions">
-      <h1>Definitions for individuals</h1>
-    </section>
+    <section id="deprecated_term_definitions">
+      <h1>Deprecated term definitions</h1>
+
+      <p class="annoy">All terms in this section are <em><strong>deprecated</strong></em>, and are only kept in this vocabulary for backward compatibility. 
+        <br>New applications should not use them.
```

Comment:
A double newline provides better readability than a single newline.
```suggestion
        <br><br>New applications should not use them.
```